### ISSUE TITLE
Error! dev_bdvd -- Problem mounting PS2 ISO

### ISSUE THREAD

**Issue by n0stalgian** [aldostools/webMAN-MOD#706]
Using a phat PS3 on Evilnat 4.88 and an external drive (128Gb) formatted to NTFS .
Most recent Webman installed FULL VERSION + PrepISO

No issues mounting PSX, PS1, PSP and PS3 ISOs (with the corresponding folders on the external drive) but for some reason I am getting a dev_bdvd error when trying with the PS2 ISOs. 

I have tried with every troubleshooting I can think of. I have allowed Webman to scan and reload XMB, restarted the console and I can see the PS2 folder with the game but when I click on the CD, the dev_bdvd error shows on the upper right corner of the screen. I have re-installed all pkgs etc and even did a system restore again just cos I thought maybe installing Multiman made things work but the PS2 ISOs cannot seem to be loaded on any other file manager. 

Please let me know if I am missing anything... I am at wit's end... 

Many thanks.

**Comment by aldostools** [comment]
PS2 games are only playable from internal HDD. Try copying the ISO to /dev_hdd0/PS2ISO

prepISO can scan PS2ISO on NTFS, but it basically copies the file to /dev_hdd0/PS2ISO when the game is mounted.

**Comment by n0stalgian** [comment]
Thanks for the speedy reply. I figured this. I tried doing it via Webman setup but I think it's not designed for a big file transfer. (Or maybe I'm just not doing it properly.) Is my only option FTP transfer? It's just it takes ages to do with a cap of 1.1Mib/s (using Filezilla). New to this sort of stuff so forgive my silly questions. Thanks very much again!

**Comment by aldostools** [comment]
There are various methods to copy content to internal HDD.
1- Use IRISMAN's file manager to copy the files from NTFS drive to the internal HDD
2- Use FTP to transfer the content
3- Mount the folder where you have the files using ps3netsrv

You can get better file transfer speed if you connect the PC and PS3 directly using a crossover ethernet cable.

Probably the first option is the easier.

**Comment by n0stalgian** [comment]
Of course, the only file manager I didn't try. Thank you so much! Every ISO is loading without any issues. You're amazing! (シ_ _ )シ

**Comment by iuegisto20** [comment]
I get the same error while using an NTFS external drive, I can still play PS2 ISOs if I transfer them manually to the internal drive from the external drive (if I use a FAT32 USB stick it transfers them automatically).

I was wondering if there is another way to launch PS2 ISOs from my external drive without having to copy them manually?

I am aware that the ISOs need to be transferred somehow to the internal drive first but it would be great if it did that automatically like it does with a FAT32 USB stick.

**Comment by blackjedi98** [comment]
i'm having the same issue webman is not copy the ps2 iso over to the hdd

**Comment by aldostools** [comment]
> i'm having the same issue webman is not copy the ps2 iso over to the hdd

Verify if the ISO file (not its content) appears listed in /dev_bdvd when you mount the PS2 game.

If it listed, it could be a bug in webMAN MOD that should be analyzed. If the ISO is not listed, it could be an issue with your setup. Try rescanning the games using prepISO.

**Comment by blackjedi98** [comment]
In the webMAN MOD UI on my PC, I see the error: "404 path not found dev_bdvd". also I'm transferring from ntfs i notice this problem, when using exfat everything worked smoothly

**Comment by aldostools** [comment]
Then it should be an issue reading your NTFS drive or mounting the ISO. exfat & ntfs are processed by webMAN MOD the same way. Could you test copying the same ISO to a pendrive formatted in NTFS?

Also try unchecking the option /dev_ntfs in /setup.ps3 then save the settings.

**Comment by blackjedi98** [comment]
I tried both a pendrive and an external Hard drive and the same results after using prepISO multiple times, i also tried  unchecking the option /dev_ntfs and restarting

**Comment by aldostools** [comment]
As I commented earlier, exFAT and NTFS processed by webMAN MOD the same way.
If it's working with exFAT, then there is something wrong in the configuration of the NTFS drive that is causing the issue.
Maybe the NTFS partition is not primary or using GPT instead of MBR.

If the problem persists, it would be better to migrate the games to exFAT drive.

**Comment by blackjedi98** [comment]
ok thanks,  I've check all that and they seem to be alright.
I've been looking around and it seems the phat ps3 has this problem a lot, another problem is that multiman and irishman doesn't seem to recognized any of the new ps2 iso even after scanning.

**Comment by aldostools** [comment]
PS2 ISO images only work from internal HDD. That's why multiMAN and IRISMAN don't detect these games.

As far as I know, webMAN MOD is the unique application that detect the PS2 ISO from external drives and copy them to the internal HDD when the game is mounted.

**Comment by blackjedi98** [comment]
looks ntfs seems to be the problem but i found a post on consolemods that states to use a usb config to make it viewable, i'll return with the results

**Comment by pitrako** [comment]
I have this problem too, when webman mod tries to copy ps2 isos from external to internal automatically with prepiso it fails with dev_bdvd error, manual copy from webman mod configuration menu to internal ps2iso folder and games works fine.
